app/controller/api/interface.js

```diff
@@ -68,6 +68,22 @@ class InterfaceController extends Controller {
     }
     ctx.fail(ctx.gettext('common.delete.fail'));
   }
+
+  async download() {
+    const ctx = this.ctx;
+    const { interfaceUniqId } = ctx.query;
+    const res = await ctx.service.interface.queryInterfaceAllInfo({ interfaceUniqId });
+
+    ctx.body = res.data;
+    ctx.set('content-type', 'application/octet-stream');
+    ctx.set('content-disposition', `attachment; filename=${res.fileName}`);
```

Comment:
```suggestion
    ctx.attachmentt(filename);
```

https://koajs.com#response-attachment-filename-